crates/biome_js_parser/src/syntax/typescript/statement.rs

```diff
@@ -34,14 +34,14 @@ fn parse_literal_as_ts_enum_member(p: &mut JsParser) -> ParsedSyntax {
         JS_NUMBER_LITERAL => {
             let err = p.err_builder("An enum member cannot have a numeric name", p.cur_range());
             p.error(err);
-            p.bump_any()
+            return Absent;
```

Comment:
I am unsure about this. Should I handle it differently?

Reply:
Yes, you must `abandon` the marker, like line 40-41